src/storage/txn/scheduler.rs

```diff
@@ -426,6 +439,7 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
             .get(priority_tag)
             .inc();
 
+        let cmd_ctx = cmd.ctx().clone();
```

Comment:
I think it's better to clone it after it fails to acquire latches to avoid unnecessary clones.